miniapp/miniapp_bt_reduction_to_band.cpp

```diff
@@ -166,34 +190,30 @@ struct BacktransformBandToTridiagMiniapp {
       double gigaflops;
       {
         const double m = mat_e_host.size().rows();
-        const double n = mat_e_host.size().cols();
+        const double n = opts.eval_idx_end;
         auto add_mul = (m - opts.b) * (m - opts.b) * n;
         gigaflops = dlaf::total_ops<T>(add_mul, add_mul) / elapsed_time / 1e9;
       }
 
       // print benchmark results
       if (0 == world.rank() && run_index >= 0) {
-        std::cout << "[" << run_index << "]"
-                  << " " << elapsed_time << "s"
-                  << " " << gigaflops << "GFlop/s"
-                  << " " << dlaf::internal::FormatShort{opts.type} << " " << mat_e_host.size() << " "
+        std::cout << "[" << run_index << "]" << " " << elapsed_time << "s" << " " << gigaflops
+                  << "GFlop/s" << " " << dlaf::internal::FormatShort{opts.type} << " "
+                  << mat_e_host.size() << " (" << 0l << ", " << opts.eval_idx_end << ") "
                   << mat_e_host.blockSize() << " " << opts.b << " " << comm_grid.size() << " "
                   << pika::get_os_thread_count() << " " << backend << std::endl;
         if (opts.csv_output) {
           // CSV formatted output with column names that can be read by pandas to simplify
           // post-processing CSVData{-version}, value_0, title_0, value_1, title_1
-          std::cout << "CSVData-2, "
-                    << "run, " << run_index << ", "
-                    << "time, " << elapsed_time << ", "
-                    << "GFlops, " << gigaflops << ", "
-                    << "type, " << dlaf::internal::FormatShort{opts.type}.value << ", "
-                    << "matrixsize, " << mat_e_host.size().rows() << ", "
-                    << "blocksize, " << mat_e_host.blockSize().rows() << ", "
-                    << "band_size, " << opts.b << ", "
-                    << "comm_rows, " << comm_grid.size().rows() << ", "
-                    << "comm_cols, " << comm_grid.size().cols() << ", "
-                    << "threads, " << pika::get_os_thread_count() << ", "
-                    << "backend, " << backend << ", " << opts.info << std::endl;
+          std::cout << "CSVData-2, " << "run, " << run_index << ", " << "time, " << elapsed_time << ", "
+                    << "GFlops, " << gigaflops << ", " << "type, "
+                    << dlaf::internal::FormatShort{opts.type}.value << ", " << "matrixsize, "
+                    << mat_e_host.size().rows() << ", " << "blocksize, " << mat_e_host.blockSize().rows()
+                    << ", " << "band_size, " << opts.b << ", " << "comm_rows, "
+                    << comm_grid.size().rows() << ", " << "comm_cols, " << comm_grid.size().cols()
+                    << ", " << "threads, " << pika::get_os_thread_count() << ", " << "backend, "
+                    << backend << ", " << opts.info << ", " << "eigenvalue index begin, " << 0l << ", "
+                    << "eigenvalue index end, " << opts.eval_idx_end << std::endl;
```

Comment:
Just wondering, did we not check the formatting of this before, or did it get reformatted for some other reason? I think we might want to do a `// clang-format off` here with the old formatting?

Reply:
Not sure. I don't think I used the bogus version. If we want one entry per line I can revert to that, and disable formatting.

Reply:
In my subjective opinion the old version with one entry per line was easier to read, so I'm in favour of that.